docs/dev/execution_configuration.md

```diff
@@ -45,41 +45,41 @@ The following configuration options are available: (the default is bold)
 - **`enableClosureCleaner()`** / `disableClosureCleaner()`. The closure cleaner is enabled by default. The closure cleaner removes unneeded references to the surrounding class of anonymous functions inside Flink programs.
 With the closure cleaner disabled, it might happen that an anonymous user function is referencing the surrounding class, which is usually not Serializable. This will lead to exceptions by the serializer.
 
-- `getParallelism()` / `setParallelism(int parallelism)` Set the default parallelism for the job.
+- `getParallelism()` / `setParallelism(int parallelism)`. Set the default parallelism for the job.
```

Comment:
The period doesn't make sense as it isn't a sentence.

Reply:
Some of the items has period some doesn't. should we remove period for the ones with?

Reply:
I would remove them.